photon-api/src/integTest/scala/com/linkedin/photon/ml/estimators/GameEstimatorIntegTest.scala

```diff
@@ -449,14 +449,14 @@ object GameEstimatorIntegTest {
     override def prepareValidationEvaluators(gameDataSet: RDD[(UniqueSampleId, GameDatum)]): Seq[Evaluator] =
       super.prepareValidationEvaluators(gameDataSet)
 
-    override def prepareNormalizationContextWrappers(dataSets: Map[CoordinateId, D forSome { type D <: DataSet[D] } ])
+    override def prepareNormalizationContextWrappers(dataSets: Map[CoordinateId, D forSome { type D <: Dataset[D] } ])
```

Comment:
We'll have to be careful about collisions with spark's `Dataset` class once we migrate to it.